modules/languages/css.nix

```diff
@@ -0,0 +1,72 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}: let
+  inherit (builtins) attrNames;
+  inherit (lib) mkEnableOption mkOption mkIf mkMerge isList types nvim;
+
+  cfg = config.vim.languages.css;
+
+  defaultServer = "vscode-langservers-extracted";
+  servers = {
+    vscode-langservers-extracted = {
+      package = pkgs.nodePackages.vscode-langservers-extracted;
+      lspConfig = ''
+        -- enable (broadcasting) snippet capability for completion
+        -- see <https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#cssls>
+        local capabilities = vim.lsp.protocol.make_client_capabilities()
```

Comment:
this might override the original definition of `capabilities`, I'm not sure where its defined but maybe configure that one globally (if it doesn't affect other LSPs), or rename this one to `css_capabilities`

Reply:
that's a good call